lib/rdstation/error/formatter.rb

```diff
@@ -30,13 +30,12 @@ def to_array
 
       def from_single_hash
         error_hash = @error_response.dup
-        error_message = error_hash.delete('error')
+        error_message = error_hash.delete('error') || error_hash.delete('message')
 
         [
           {
             'error_type' => 'TOO_MANY_REQUESTS',
-            'error_message' => error_message,
-            'details' => error_hash
```

Comment:
Because it won't be used anymore, who will return 429 from now will be the API Gateway, and this info is going to be in the headers, accessing the `.details` from the raised exception.